### ISSUE TITLE
Can't resolve 'isomorphic-form-data'

### ISSUE THREAD

**Issue by chaliy** [backstage/backstage#6619]
Going though [Getting Started](https://backstage.io/docs/getting-started/create-an-app) produce application that fail to start with `Can't resolve 'isomorphic-form-data' `

## Expected Behavior

```yarn dev``` starts application

## Current Behavior

```yarn dev``` starts application, but frontend application fails to load with error message:

```
[0] ERROR in /U....e/node_modules/swagger-ui-react/swagger-ui.js
[0] Module not found: Error: Can't resolve 'isomorphic-form-data' in '/U...e/node_modules/swagger-ui-react'
[0] ℹ ｢wdm｣: Failed to compile.
```

## Steps to Reproduce

1. `npx @backstage/create-app`
2. `yarn dev`
3. open localhost:3000

## Your Environment

- NodeJS Version:  v14.17.3
- Operating System and Version: Mac
- Browser Information: Google Chrome
- swagger-ui-react: 3.51.1

**Comment by chaliy** [comment]
Executing `lerna add isomorphic-form-data  packages/app` after `npx @backstage/create-app` fixes an issue.

**Comment by benjdlambert** [comment]
Yep - looks like there was an issue with the most recent release of `swagger-ui` we fixed it locally here #6626 but that won't be released until thursday, so will keep this around for reference for users that might come across this.

**Comment by benjdlambert** [comment]
Closing this as it should be fixed with the release that went out yesterday. Please feel free to re-open if it is not.
